src/main.js

```diff
@@ -44,14 +44,83 @@ const RESPONSE = {
 
+const RESPONSE2 = {
+    'issues': [{
+        'flags': [
+            [0, 5, 'Beginning', 'Flags should work at the beginning of the text',
+                'See if this flag works', -1.0
+            ],
+            [18, 22, 'Middle', 'Flags should work in the middle of the text',
+                'See if this flag works', -1.0
+            ],
+            [50, 53, 'End', 'Flags should work at the end of the text',
+                'See if this flag works', -1.0
+            ]
+        ],
+        'name': 'Flag position test',
+        'summary': 'Flags should work correctly regardles of where they appear in the text.'
+    }],
+    'text': 'Start. There is a flag in the middle of the text. End.'
+};
+
+
+
+const RESPONSE3 = {
+    'issues': [{
+        'flags': [
+            [7, 14, 'Beginning', 'Flags should work at the beginning of the text',
+                'See if this flag works', -1.0
+            ],
+            [21, 28, 'Middle', 'Flags should work in the middle of the text',
+                'See if this flag works', -1.0
+            ],
+            [37, 44, 'End', 'Flags should work at the end of the text',
+                'See if this flag works', -1.0
+            ]
+        ],
+        'name': 'Flag position test',
+        'summary': 'Flags should work correctly regardles of where they appear in the text.'
+    }],
+    'text': 'She is willing. I am willing. She is willing.'
+};
+
+// const RESPONSE4 = {
+//     'issues': [{
+//         'flags': [
+//             [8, 15, 'Beginning', 'Flags should work at the beginning of the text',
+//                 'See if this flag works', -1.0
+//             ],
+//             [27, 34, 'Middle', 'Flags should work in the middle of the text',
+//                 'See if this flag works', -1.0
+//             ],
+//             [44, 51, 'Middle', 'Flags should work at the end of the text',
+//                 'See if this flag works', -1.0
+//             ],
+//             [67, 74, 'End', 'Flags should work at the end of the text',
+//                 'See if this flag works', -1.0
+//             ]
+//         ],
+//         'name': 'Flag position test',
+//         'summary': 'Flags should work correctly regardles of where they appear in the text.'
+//     }],
+//     'text': 'She is  willing to         willing          willing,               willing.'
+// };
+
 if (process.env.NODE_ENV === "test") {
     new Server({
 
         routes() {
             this.timing = 0;
             this.urlPrefix = 'https://api.biascorrect.org';
-            this.post("/check", () => {
-                return RESPONSE;
+            this.post("/check", (schema, request) => {
+                let text = JSON.parse(request.requestBody).text;
+                if (text.includes('willing')) {
+                    return RESPONSE1;
+                } else if (text.includes('She')) {
+                    return RESPONSE3;
+                } else {
+                    return RESPONSE2;
+                }
```

Comment:
Glad you got this to work!

The responses should have meaningful names, instead of RESPONSE1, 2, 3.